0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -36,67 +36,34 @@ The server used on the recipient's side to handle SPSP requests.
 
 #### Receiver
 
-The specific HTTP endpoint on the SPSP used for setting up a payment.
+The specific HTTPS endpoint on the SPSP used for setting up a payment.
```

Comment:
This "definitions" section might read better as a list of bullets with bolded terms rather than extra sub-headers, e.g.

- **SPSP Client:** The sender application that uses SPSP to interact with the SPSP Server.
- **SPSP Server:** The server used on the recipient's side to handle SPSP requests.
- **Receiver:** The specific HTTPS endpoint on the SPSP used for setting up a payment.

Also, I think the Receiver definition is missing a word here: "on the SPSP _Server_" probably. I'd reword the sentence overall:

> **Receiver:** The HTTPS endpoint provided by the SPSP Server, which the SPSP Client calls to set up a payment.